pkg/collector/corechecks/system/disk/disk/disk.go

```diff
@@ -149,34 +268,431 @@ func (c *Check) instanceConfigure(data integration.Data) error {
+			if re, err := regexp.Compile(strVal); err == nil {
+				c.cfg.excludedFilesystems = append(c.cfg.excludedFilesystems, *re)
+			} else {
+				return err
+			}
+		}
+	}
+	for _, key := range []string{"file_system_exclude", "file_system_blacklist", "excluded_filesystems"} {
+		if fileSystemExclude, ok := instanceConfig[key].([]interface{}); ok {
+			for _, val := range fileSystemExclude {
+				if strVal, ok := val.(string); ok {
+					regexp, err := regexp.Compile(strVal)
+					if err != nil {
+						return err
+					}
+					c.cfg.excludedFilesystems = append(c.cfg.excludedFilesystems, *regexp)
+				}
+			}
+		}
+	}
+	return nil
+}
+
+func (c *Check) configureIncludeFileSystem(conf map[interface{}]interface{}) error {
+	for _, key := range []string{"file_system_include", "file_system_whitelist"} {
+		if fileSystemInclude, ok := conf[key].([]interface{}); ok {
+			for _, val := range fileSystemInclude {
+				if strVal, ok := val.(string); ok {
+					regexp, err := regexp.Compile(strVal)
+					if err != nil {
+						return err
+					}
+					c.cfg.includedFilesystems = append(c.cfg.includedFilesystems, *regexp)
+				}
+			}
+		}
+	}
+	return nil
+}
+
+func (c *Check) configureExcludeMountPoint(instanceConfig map[interface{}]interface{}, initConfig map[interface{}]interface{}) error {
+	defaultMountPoints := []interface{}{"(/host)?/proc/sys/fs/binfmt_misc$"}
+	var mountPointExclude []interface{}
+	for _, key := range []string{"mount_point_global_exclude", "mount_point_global_blacklist"} {
+		if val, ok := initConfig[key].([]interface{}); ok {
+			mountPointExclude = val
+			break // Stop searching after the first valid key is found
+		}
+	}
+	// Use default values if neither key was found
+	if mountPointExclude == nil {
+		mountPointExclude = defaultMountPoints
+	}
+	for _, val := range mountPointExclude {
+		if strVal, ok := val.(string); ok {
+			if re, err := regexp.Compile(strVal); err == nil {
+				c.cfg.excludedMountpoints = append(c.cfg.excludedMountpoints, *re)
+			} else {
+				return err
+			}
+		}
+	}
+	for _, key := range []string{"mount_point_exclude", "mount_point_blacklist"} {
+		if mountPointExclude, ok := instanceConfig[key].([]interface{}); ok {
+			for _, val := range mountPointExclude {
+				if strVal, ok := val.(string); ok {
+					regexp, err := regexp.Compile(strVal)
+					if err != nil {
+						return err
+					}
+					c.cfg.excludedMountpoints = append(c.cfg.excludedMountpoints, *regexp)
+				}
+			}
+		}
+	}
+	excludedMountPointRe, found := instanceConfig["excluded_mountpoint_re"]
+	if excludedMountPointRe, ok := excludedMountPointRe.(string); found && ok {
+		var err error
+		regexp, err := regexp.Compile(excludedMountPointRe)
+		if err != nil {
+			return err
+		}
+		c.cfg.excludedMountpoints = append(c.cfg.excludedMountpoints, *regexp)
+	}
+	return nil
+}
+
+func (c *Check) configureIncludeMountPoint(conf map[interface{}]interface{}) error {
+	for _, key := range []string{"mount_point_include", "mount_point_whitelist"} {
```

Comment:
For the record I believe the Python version only uses one of them, not both (if they are both provided)
I do think we can use both here as it's just more intuitive

Same issue with some other configurations